src/main/java/org/humancellatlas/ingest/security/SecurityConfig.java

```diff
@@ -72,19 +87,28 @@ protected void configure(HttpSecurity http) throws Exception {
                 .sessionManagement().sessionCreationPolicy(SessionCreationPolicy.STATELESS).and()
                 .cors().and()
                 .authorizeRequests()
-                .antMatchers(HttpMethod.POST, "/submissionEnvelopes").authenticated()
-                .antMatchers(HttpMethod.POST, "/submissionEnvelopes/*/projects").authenticated()
-                .antMatchers(HttpMethod.POST, "/projects**").authenticated()
-                .antMatchers(GET, "/projects").hasAuthority(WRANGLER.name())
-                .antMatchers(POST, "/auth/registration").hasAuthority(GUEST.name())
+                .antMatchers(POST, "/submissionEnvelopes").authenticated()
+                .antMatchers(POST, "/projects").authenticated()
+                .antMatchers(GET, "/user/**").authenticated()
                 .antMatchers(GET, "/auth/account").authenticated()
-                .requestMatchers(this::isRequestForSecuredResourceFromProxy).authenticated()
+                .antMatchers(POST, "/auth/registration").hasAuthority(GUEST.name())
+                .requestMatchers(this::isSecuredWranglerEndpointFromOutside).hasAnyAuthority(WRANGLER.name(), SERVICE.name())
+                .requestMatchers(this::isSecuredEndpointFromOutside).authenticated()
                 .antMatchers(GET, "/**").permitAll();
     }
 
-    private Boolean isRequestForSecuredResourceFromProxy(HttpServletRequest request) {
+    private Boolean isSecuredEndpointFromOutside(HttpServletRequest request) {
         return SECURED_ANT_PATHS.stream().anyMatch(matcher -> matcher.matches(request)) &&
-                Optional.ofNullable(request.getHeader(FORWARDED_FOR)).isPresent();
+                this.isRequestOutsideProxy(request);
+    }
+
+    private Boolean isSecuredWranglerEndpointFromOutside(HttpServletRequest request) {
+        return SECURED_WRANGLER_ANT_PATHS.stream().anyMatch(matcher -> matcher.matches(request)) &&
+                this.isRequestOutsideProxy(request);
+    }
+
+    private Boolean isRequestOutsideProxy(HttpServletRequest request) {
+        return Optional.ofNullable(request.getHeader(FORWARDED_HOST)).isPresent();
```

Comment:
I think all of these can be made static, which would make them easier to test since they wouldn't require object instantiation and associated dependency mocking